lib/utils/cli.go

```diff
@@ -140,9 +140,44 @@ func GetIterations() int {
 	return iter
 }
 
-// UserMessageFromError returns user friendly error message from error
+// UserMessageFromError returns user-friendly error message from error.
+// The error message will be formatted for output depending on the debug
+// flag
 func UserMessageFromError(err error) string {
-	// untrusted cert?
+	if err == nil {
+		return ""
+	}
+	if certErr := formatCertError(err); certErr != "" {
+		return certErr
+	}
+	if log.GetLevel() == log.DebugLevel {
```

Comment:
In debug mode, shouldn't we return the human readable error message as well as the stack trace?

Reply:
Sure - it could but this was an attempt to also mitigate [this](https://github.com/gravitational/teleport/issues/3205) since the error message incl. debug stack trace popping up in the intermediary (i.e. proxy) is not going to look readable.